docs/contents/components/overlay/modal/accessibility.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Modal
+description: "`Modal` is a component that is displayed over the main content to focus the user's attention solely on the information."
+package-name: "@yamada-ui/modal"
+with_description: true
+tab: Accessibility
+---
+
+`Modal` follows the [WAI-ARIA - Dialog (Modal) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) pattern.
+
```

Comment:
The sentence repeats the word "Pattern", is that alright?

Reply:
Maybe it could be like below.
```suggestion

The `Modal` follows the [WAI-ARIA - Dialog (Modal) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) for accessibility.

```